### PR TITLE
Dev/remove sensor altitude

### PR DESCRIPTION
**Pull request by linus-sherrill**
Rationalize sensor altitude in metadata.

Combined SENSOR_ALTITUDE into SENSOR_LOCATION by leveraging 3D geo point. Removed SENSOR_ALTITUDE metadata tag since it now redundant. This change removed warning about unreferenced altitude.

Also consolidated target location

Reformatted the metadata tags table in a separate commit so white-space changes do not overshadow the more substantial changes.

### CODE REVIEW

File: vital/klv/convert_0104_metadata.cxx [hunk 1]
```diff
@@ -145,7 +155,7 @@ ::convert_0104_metadata( klv_uds_vector_t const& uds, metadata& md )
   // lat-lon points and image corner points. All geodetic points are assumed to
   // be WGS84 lat-lon.
```
Comment (aaron-bray) [inline vital/klv/convert_0104_metadata.cxx]: We should specify the units we are using in these description comments (degrees and meters)

File: vital/types/metadata_tags.h [hunk 2]
```diff
@@ -48,95 +48,93 @@
 //      tag                          string name                        type
 //      ---                          -----------                        ----
 #define KWIVER_VITAL_METADATA_TAGS(CALL)                                \
```
Comment (aaron-bray) [inline vital/types/metadata_tags.h]: It bugs me there are no units here, not sure if that is documented somewhere else...

**Comment by aaron-bray** [comment]
Besides unit documentation, the organization looks good to me!

File: vital/types/metadata_tags.h (hunk 2 above)
Comment (linus-sherrill) [reply vital/types/metadata_tags.h]: I could add a more extended description to some of these items (and risk the wrath of the line length enforcers). Additional description could also be helpful for some items. I could add an additional field with an extended description.

File: vital/types/metadata_tags.h [hunk 3]
```diff
@@ -48,95 +48,93 @@
...
+CALL( SENSOR_REL_EL_ANGLE,         "Sensor Relative Elevation Angle",      double ) \
+CALL( SENSOR_REL_ROLL_ANGLE,       "Sensor Relative Roll Angle",           double ) \
+CALL( SENSOR_YAW_ANGLE,            "Sensor yaw angle",                     double ) \
+CALL( SENSOR_PITCH_ANGLE,          "Sensor pitch angle",                   double ) \
+CALL( SENSOR_ROLL_ANGLE,           "Sensor Roll Angle",                    double ) \
+CALL( SENSOR_TYPE,                 "Sensor Type",                          std::string ) \
+CALL( SLANT_RANGE,                 "Slant Range",                          double ) \
+CALL( TARGET_WIDTH,                "Target Width",                         double ) \
+CALL( FRAME_CENTER,                "Geodetic Frame Center, Lat/Lon",       geo_point ) \
+CALL( FRAME_CENTER_ELEV,           "Frame Center Elevation",               double ) /* TODO: merge with previous? */ \
```
Comment (mleotta) [inline vital/types/metadata_tags.h]: What about this case?  Should we merge this with the previous as well?

**Comment by linus-sherrill** [comment]
Jenkins, test this please

**Pull request merged**